From bc9c23a4350598db4ec04225e1e0194a7b17175d Mon Sep 17 00:00:00 2001
From: Jim Meyering <meyering@fb.com>
Date: Sun, 27 Oct 2013 01:34:03 +0000
Subject: maint: avoid automake deprecation warning re ACLOCAL_AMFLAGS

* Makefile.am (ACLOCAL_AMFLAGS): Don't use this deprecated variable.
* configure.ac (AC_CONFIG_MACRO_DIRS): Use this instead.
(AUTOMAKE_OPTIONS): Require automake-1.12.
---
diff --git a/Makefile.am b/Makefile.am
index d2ba6e6..8090826 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -15,8 +15,7 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
-AUTOMAKE_OPTIONS = gnu 1.10
-ACLOCAL_AMFLAGS = -I m4
+AUTOMAKE_OPTIONS = gnu 1.12
 
 SUBDIRS = po lib doc src tests gnulib-tests
 
diff --git a/configure.ac b/configure.ac
index 1ebc253..b8d55af 100644
--- a/configure.ac
+++ b/configure.ac
@@ -66,6 +66,7 @@ AC_CONFIG_AUX_DIR(build-aux)
 AC_CONFIG_SRCDIR(src/grep.c)
 AC_DEFINE([GREP], 1, [We are building grep])
 AC_PREREQ(2.59)
+AC_CONFIG_MACRO_DIRS([m4])
 
 dnl Automake stuff.
 AM_INIT_AUTOMAKE([1.11 no-dist-gzip dist-xz color-tests parallel-tests
--
cgit v0.9.0.2
